launchable/commands/record/tests.py

```diff
@@ -241,7 +241,7 @@ def tests(
             raise e
         else:
             traceback.print_exc()
-            return
+            exit(0)
```

Comment:
If we exit here, we do not need the following procedure anymore. Can you please remove it?

https://github.com/launchableinc/cli/blob/main/launchable/test_runners/launchable.py#L108-L112